jwst/tweakreg/wcsimage.py

```diff
@@ -62,8 +62,9 @@ def __init__(self, wcs, v2_ref, v3_ref, roll_ref, ra_ref, dec_ref):
             DEC of the reference point in degrees.
 
         """
-        if not self._check_wcs_structure(wcs):
-            raise ValueError("Unsupported WCS structure.")
+        valid, message =  self._check_wcs_structure(wcs)
+        if not valid:
+            raise ValueError(message)
```

Comment:
If you insist on having a more detailed error message, I would prefer that it be like this: `raise ValueError("Unsupported WCS structure: " + message)`